src/csharp/Directory.Build.props

```diff
@@ -34,6 +34,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)..\..\LICENSE"
+         Pack="true"
```

Comment:
(nit) it is nice to have the attributes lined up vertically with the attribute on the first line. See the `<Content` xml below for an example.

Reply:
Fixed.